### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please note that corpus has been renamed to testsuite. Also, you don't need to run git clone here.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+image: ghcr.io/capuanob/gifh-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Tests -> testsuite. You may omit the image name.

File: fuzz.cc [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#include <cstdio> // FILE, fmemopen
```
Comment (xansec) [inline fuzz.cc]: Can we rename this to indicate what's being fuzzed?

File: gif.h [hunk 4]
```diff
@@ -732,6 +732,61 @@ typedef struct
     bool firstFrame;
```
Comment (xansec) [inline gif.h]: Why do we include this in the existing gif.h header? Can we create a separate header to store fuzzer specific function calls?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I have made the requested changes, and look forward to your feedback!

File: gif.h [hunk 5]
```diff
@@ -732,6 +732,7 @@ typedef struct
...
```
Comment (xansec) [inline gif.h]: Double check your formatting - this file is still showing up as modified...

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Review by xansec — APPROVED** [review]
I was able to get the header out of the PR. Merging. LGTM!

**Pull request merged**